src/actions/write/increaseVault.ts

```diff
@@ -12,7 +12,7 @@ import { safeIncreaseVault } from '../../vaults';
 // TODO: use top level class
 export class IncreaseVault {
   id: TransactionId;
-  qty: PositiveFiniteInteger; // TODO: change to IO Token
+  qty: mIOToken; // TODO: change to IO Token
```

Comment:
to done

Reply:
[60a3aa7](https://github.com/ar-io/testnet-contract/pull/214/commits/60a3aa700c50a8887bbd557315c1d5b263ecc166)